partner_address_street3/hooks.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Nicolas Bessi, Alexandre Fayolle, Camptocamp SA
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def post_init_hook(cr, registry):
+    query = """
+        UPDATE res_country
+        SET address_format = replace(
+        address_format,
+        E'%(street2)s\n',
+        E'%(street2)s\n%(street3)s\n'
```

Comment:
added it in https://github.com/OCA/partner-contact/pull/316/files#diff-f9fdb975d77b348b5bab55bc71ebf13bR14